test/CSParametersRegistry.t.sol

```diff
@@ -1031,3 +1037,34 @@ contract CSParametersRegistryStrikesParamsTest is CSParametersRegistryBaseTest {
         assertEq(thresholdOut, defaultInitData.strikesThreshold);
     }
 }
+
+contract CSParametersRegistryBadPerformancePenaltyTest is
```

Comment:
Pls, implement tests for the missing methods. Take the contracts above as an example